snap/snapcraft.yaml

```diff
@@ -13,13 +13,17 @@ confinement: strict
 
 parts:
     novnc:
-        source: build/
+        source: https://github.com/novnc/noVNC/archive/v1.2.0.tar.gz
```

Comment:
We want it to build the working directory, not something else.

Reply:
I always thought the snap be built from what's released, i.e 1.2.0 release in this case. Is that 'build' a working directory on your computer? I guess either way, as long as it works then all good

Reply:
On the computer building the snap, yes. I'd say that is more expected than the snap pulling down something else and packaging.

Reply:
From your comment down below I see you're using Github workflows to do the build, so that build/ directory is something created on the VM running on github's servers when the publish workflow is running.
That's fine, but it means no one else can build the snap, so it'd be up to you to re-run that workflow to fix it when the changes have been made